getting-started-with-directus-and-flask/index.md

```diff
@@ -0,0 +1,396 @@
+	<body>
+		<header class="container">
+			<nav>
+				<ul>
+					<li>
+						<a href="/"><strong>Home</strong></a>
+					</li>
+				</ul>
+				<ul>
+					<li><a href="/blog">Blog</a></li>
+					<li><a href="/about">About</a></li>
+					<li><a href="/conduct">Code of Conduct</a></li>
+					<li><a href="/privacy">Privacy Policy</a></li>
+				</ul>
+			</nav>
+		</header>
+
+		<main class="container">{% block content %}{% endblock %}</main>
+	</body>
+</html>
+```
+
+:::info
+
+Note there is a stylesheet in this code that imports [Pico](https://picocss.com/), a minimal class-less CSS library used to add some basic styles on the website. It was only used on this article to make things look better, but you can feel free to use whichever library you prefer and even use the default browser styles.
+
+:::
+
+Now create a *templates/home.html* file that will extend the base template and display some data on it. This file will have the following content:
+
+```jinja
+{% extends "base.html" %}
+
+{% block content %}
+<article>
+	<h1>{% block title %}{{ title }}{% endblock %}</h1>
+	<p>{{ description }}</p>
+</article>
+{% endblock %}
+```
+
+### Updating Home Route
+
+After creating the templates, update the *app.py* file:
+
+```py
+from flask import Flask // [!code --]
+from flask import Flask, render_template // [!code ++]
+import directus // [!code ++]
+
+app = Flask(__name__)
+
+
+@app.get("/")
+def home():
+    return "Hello world" // [!code --]
+    global_data = directus.get_global_data() // [!code ++]
+// [!code ++]
+    return render_template( // [!code ++]
+        "home.html", title=global_data["title"], description=global_data["description"] // [!code ++]
+    ) // [!code ++]
+```
+
+Then go to [http://localhost:3000](http://localhost:3000) and you will see a page like this:
+
+![Home page displaying configured global data](home-page.png)
+
+
+## Creating Pages With Directus
+
+Note that the base template has some links on its navbar pointing to pages like **About** that typically have just some text content and some images, but also need to be updated frequently and by people who do not have access to the code base.
+
+Using Directus you can make these pages dynamically available on your site and editable by non-technical users.
+
+To accomplish this, create a new collection called `pages` - make the Primary ID Field a "Manually Entered String" called `slug`, which will correlate with the URL for the page. For example, `about` will later correlate to the page `localhost:3000/about`.
+
+Create a text input field called `title` and a WYSIWYG input field called `content`. In Access Control, give the Public role read access to the new collection. Create 3 items in the new collection - [here's some sample data](https://github.com/directus-community/getting-started-demo-data).
+
+### Rendering Dynamic Pages
+
+To get data of pages registered on the `pages` collection you will need to add the following code at the end of the *directus.py* file:
```

Comment:
```suggestion
To get data of pages registered on the `pages` collection you will need to add the following code at the end of the `directus.py` file:
```

I'm going to stop changing filenames from bold to inline code - please can you do this throughout the remainder of the post?